task_2.py

```diff
@@ -1,2 +1,7 @@
-def chess_board(n, k):
-    pass
\ No newline at end of file
+def chess_board(n,k):
+    for p in range(0,2 * n):
+        for i in range(0,2):
```

Comment:
Zauważ, że chcesz w tej pętli tyle obrotów ile wynosi rozmiar pola szachownicy - `for _ in range(k)`